### PR TITLE
update readme and delete stale content

### PR DESCRIPTION
**Pull request by jeffxtang**
# What does this PR do?

Fixes # (issue)

## Feature/Issue validation/testing

Please describe the tests that you ran to verify your changes and relevant result summary. Provide instructions so it can be reproduced.
Please also list any relevant details for your test configuration.

- [ ] Test A
Logs for Test A

- [ ] Test B
Logs for Test B

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/facebookresearch/llama-recipes/blob/main/CONTRIBUTING.md#pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes?  
- [ ] Did you write any new necessary tests?

Thanks for contributing 🎉!

### CODE REVIEW

File: recipes/inference/llama_web_ui/Llama2_Gradio.ipynb
Comment (WuhanMonkey) [inline recipes/inference/llama_web_ui/Llama2_Gradio.ipynb:1]: @jeffxtang, did you just delete the entire llama web ui folder? Why?

Comment (jeffxtang) [reply recipes/inference/llama_web_ui/Llama2_Gradio.ipynb:1]: yes @WuhanMonkey that folder https://github.com/meta-llama/llama-recipes/tree/main/recipes/inference/llama_web_ui is old and too simple and not worth updating to llama 3: gradio is covered by your notebook, and i'll add a streamlit one for real rag case soon.

**Review by WuhanMonkey — APPROVED** [review]
Just wanna confirm if you are deleting entire llama_web_ui folder. If so, may consider it in a separate PR? But your choice!

**Comment by jeffxtang** [comment]
> Just wanna confirm if you are deleting entire llama_web_ui folder. If so, may consider it in a separate PR? But your choice!

thanks for confirming. i'll just kill 2 birds w/ 1 PR this time...

**Pull request merged**